jsonrpc/server.go

```diff
@@ -187,9 +187,16 @@ func (s *Server) registerMethod(method Method) error {
 	return nil
 }
 
+type connKey struct{}
```

Comment:
We get this context from http package, it might me carrying other values already. can we use something with more context, to avoid collusion?

"jsonrpc.ctx.connection" or something?

Reply:
Collision is impossible https://brabalawuka.cc/posts/study/empty_struct/

I can update the name if you think it's unclear, though.

Reply:
Wow, nice. TIL.
Thanks!